mobile/src/main/java/com/dexdrip/stephenblack/nightwatch/integration/DexdripDataReceiver.java

```diff
@@ -0,0 +1,17 @@
+package com.dexdrip.stephenblack.nightwatch.integration;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.content.WakefulBroadcastReceiver;
+
+/**
+ * Tells us when G4 data has been updated.
+ */
+public class DexDripDataReceiver extends WakefulBroadcastReceiver {
```

Comment:
Due to the Drip being capitalized here but not in the file name, android studio complains about this class, mind adjusting one or the other?